bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -115,11 +115,19 @@ <h2 class="primary go">{{ _('Discover <br>what drives our mission') }}</h2>
           </a>
         </li>
       {% endif %}
+      {% if waffle.switch('mozorg-home-privacy-promo') and l10n_has_tag('privacy_day') %}
```

Comment:
micro-optimization nit: put the l10n_has_tag check in front of the waffle.switch to short-circuit a potentially unnecessary DB query.